TodoApp/TodoApp.Api.Tests/Repositories/TodoRepository_Tests.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using TodoApp.Api.Models;
+using TodoApp.Api.Repositories;
+
+namespace TodoApp.Api.Tests.Repositories
+{
+    [TestFixture]
+    class TodoRepository_Tests
+    {
+        private IAsyncTodoRepository _repository;
+
+        public TodoRepository_Tests()
```

Comment:
Convert this to SetUp method == method with `[SetUp]` attribute that run before each test. This should fix the issues with the dependent tests.